HomeTask2/Task2_2.go

```diff
@@ -0,0 +1,18 @@
+package main
+
+import "fmt"
+
+func max(sl []string) string {
+	word := sl[0]
+	for _, a := range sl {
+		if len(a) > len(word) {
+			word = a
+		}
+	}
+	return word
+}
+
+func main() {
```

Comment:
Usually it's not the best idea to have multiply `main` funcs in one package. In fact, it'll be a conflict.
Though, it'll work while running `go run filename.go`, but not advised.